frontend/components/details/ImageDetail.tsx

```diff
@@ -0,0 +1,24 @@
+import { ImageModule } from "@/interfaces";
+import { FC } from "react";
+import ImageCard from "../cards/ImageCard";
+
+interface ImageDetailProps {
+  image: ImageModule;
+}
+
+const ImageDetail: FC<ImageDetailProps> = ({ image }: ImageDetailProps) => {
+  const firstImage = image["serializedImages"]?.[0];
+  return (
+    <>
+      <h1 className="text-center card-title" dangerouslySetInnerHTML={{ __html: image["name"] }} />
```

Comment:
Do images have a `name`? I can't find where that's coming from.
https://github.com/ModularHistory/modularhistory/blob/5b6ae545f3c62904f58d71f7e7b37e7636363324/apps/images/models/image.py#L51
https://github.com/ModularHistory/modularhistory/blob/6f102cd1e165f2a00240a649b0e6b779244a9507/apps/images/api/types.py#L7

Reply:
Fixed.